www/.nvmrc

```diff
@@ -0,0 +1 @@
+0.10.33
```

Comment:
Nah, a lot of modules break on 0.12.x -- let's defer upgrading to 0.12.x for a little while.  Also io.js.